src/seq/mod.rs

```diff
@@ -1192,23 +1260,23 @@ mod test {
                 chunk_remaining: 32,
                 hint_total_size: true,
             }),
-            Some(40)
+            Some(27)
         );
         assert_eq!(
             choose(WindowHintedIterator {
                 iter: 0..100,
                 window_size: 32,
                 hint_total_size: false,
             }),
-            Some(40)
+            Some(27)
```

Comment:
Curious why these all return the same result. Different hinting should affect this?

Reply:
This is `choose_stable` which is supposed to return the same result regardless of size hints. It reassuringly also returns the same result as the unhinted iterator.